src/output-json-dns.c

```diff
@@ -323,6 +323,7 @@ static int JsonDnsLoggerToServer(ThreadVars *tv, void *thread_data,
         }
 
         jb_open_object(jb, "dns");
+        jb_set_int(jb, "version", 2);
```

Comment:
Should the one in `dns_log_json_answer ` be moved before the call of `rs_dns_log_json_answer` ?

Reply:
Is there a reason why?

Reply:
Consistency ?..

Let's go your way